modules/nf-core/porechop/abi/main.nf

```diff
@@ -0,0 +1,39 @@
+process PORECHOP_ABI {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::porechop_abi=0.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/porechop_abi:0.5.0--py310h590eda1_0':
+        'quay.io/biocontainers/porechop_abi:0.5.0--py310h590eda1_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val abi
+
+    output:
+    tuple val(meta), path("*.fastq.gz"), emit: reads
+    tuple val(meta), path("*.log")     , emit: log
+    path "versions.yml"                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def abi_initio = abi ? " -abi " : ''
+    """
+    porechop_abi \\
+        $abi_initio \\
+        -i $reads \\
+        -t $task.cpus \\
+        $args \\
+        -o ${prefix}.fastq.gz \\
```

Comment:
Are there any long forms available for the arguments? for example `-i` is `--input` in some modules.

Reply:
Yes, from `porechop_abi --help`


```
Main options:
  -i INPUT, --input INPUT               FASTA/FASTQ of input reads or a directory which will be recursively searched for FASTQ files (required)
  -o OUTPUT, --output OUTPUT            Filename for FASTA or FASTQ of trimmed reads (if not set, trimmed reads will be printed to stdout)
  --format {auto,fasta,fastq,fasta.gz,fastq.gz}
                                        Output format for the reads - if auto, the format will be chosen based on the output filename or the input read format (default: auto)
  -v VERBOSITY, --verbosity VERBOSITY   Level of progress information: 0 = none, 1 = some, 2 = lots, 3 = full - output will go to stdout if reads are saved to a file and stderr if reads are printed to stdout (default: 1)
  -t THREADS, --threads THREADS         Number of threads to use for adapter alignment (default: 8)

```

Reply:
Ah can you change it to the long form? It improves the readability :)

Reply:
Great, I will do it for all options :)